src/main/java/org/openrewrite/java/spring/framework/MigrateHandlerInterceptor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 the original author or authors.
+ * Copyright 2024 the original author or authors.
```

Comment:
No need to update these copyright headers; only for new recipes do we want those to reflect the year they were added.